### PR TITLE
Clean(platform): Clean appKey method and usage

### PR DESCRIPTION
**Pull request by hyahiaoui-ext**
## Description

AppKey was introduced (18.10) to give an unique ID to a platform (but we use uuid now)
It is not used anymore and could be removed.

**Fixes** # MON-12739

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [x] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.04.x
- [ ] 21.10.x
- [x] 22.04.x
- [x] 22.10.x (master)

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by jeremyjaouen — DISMISSED** [review]

**Review by jeremyjaouen — DISMISSED** [review]

**Review by callapa — CHANGES_REQUESTED** [review]
You forgot to clean up this file: src/CentreonRemote/Application/Webservice/CentreonRemoteServer.php

**Comment by hyahiaoui-ext** [comment]
> You forgot to clean up this file: src/CentreonRemote/Application/Webservice/CentreonRemoteServer.php

Done :) @callapa

File: www/install/php/Update-22.10.0-beta.1.php [hunk 1]
```diff
@@ -32,6 +32,7 @@
 
     $errorMessage = "Unable to delete 'oreon_web_path' option from database";
     $pearDB->query("DELETE FROM `options` WHERE `key` = 'oreon_web_path'");
+    $pearDB->query("DELETE FROM `informations` WHERE `key` = 'appKey'");
```
Comment (kduret) [inline www/install/php/Update-22.10.0-beta.1.php:37]: please add a related error message

**Review by kduret — COMMENTED** [review]
remove `remote_servers.app_key` column from fresh install and upgrade

File: www/install/php/Update-22.10.0-beta.1.php (hunk 1 above)
Comment (hyahiaoui-ext) [reply www/install/php/Update-22.10.0-beta.1.php:37]: @kduret done !

**Comment by hyahiaoui-ext** [comment]
> remove `remote_servers.app_key` column from fresh install and upgrade

I've made changes now you can check ^^

File: www/install/sql/centreon/Update-DB-22.10.0-beta.1.sql [hunk 2]
```diff
@@ -122,4 +122,6 @@ ALTER TABLE cfg_nagios MODIFY `enable_predictive_service_dependency_checks` enum
 ALTER TABLE cfg_nagios MODIFY `debug_verbosity` enum('0','1');
 
 
-ALTER TABLE contact DROP COLUMN IF EXISTS `enable_one_click_export`;
\ No newline at end of file
+ALTER TABLE contact DROP COLUMN IF EXISTS `enable_one_click_export`;
+
+ALTER TABLE remote_servers DROP COLUMN IF EXISTS `app_key`;
```
Comment (kduret) [inline www/install/sql/centreon/Update-DB-22.10.0-beta.1.sql]: `DROP COLUMN IF EXISTS` does not work for `MySQL`
```suggestion
ALTER TABLE remote_servers DROP COLUMN `app_key`;
```

File: www/install/php/Update-22.10.0-beta.1.php [hunk 3]
```diff
@@ -32,6 +32,8 @@
 
     $errorMessage = "Unable to delete 'oreon_web_path' option from database";
     $pearDB->query("DELETE FROM `options` WHERE `key` = 'oreon_web_path'");
+    $errorMessage = "Unable to delete 'appKey' option from database";
```
Comment (kduret) [inline www/install/php/Update-22.10.0-beta.1.php]:
```suggestion

    $errorMessage = "Unable to delete 'appKey' option from database";
```

File: www/install/sql/centreon/Update-DB-22.10.0-beta.1.sql (hunk 2 above)
Comment (kduret) [reply www/install/sql/centreon/Update-DB-22.10.0-beta.1.sql]: I will create a ticket to fix query about `enable_one_click_export` (cc @a-launois )

**Review by kduret — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]

**Review by callapa — APPROVED** [review]

**Pull request merged**